core/src/main/java/com/capitalone/dashboard/repository/CustomRepositoryQuery.java

```diff
@@ -17,4 +19,6 @@
     List<Component> findComponents(Collector collector, CollectorItem collectorItem);
     List<Component> findComponents(ObjectId collectorId, CollectorType collectorType, CollectorItem collectorItem);
     List<Component> findComponents(ObjectId collectorId, CollectorType collectorType, ObjectId collectorItemId);
+    Page<CollectorItem> findByCollectorIdInAndJobNameContainingAndNiceNameContainingAllIgnoreCase(List<ObjectId> collectorId, String jobName, String niceName, Pageable pageable);
+    Page<CollectorItem> findByCollectorIdInAndJobNameContainingIgnoreCase(List<ObjectId> collectorId, String jobName,Pageable pageable);
```

Comment:
We dont want these queries in CustomRepositoryQuery class. Please move them to the respective class